index.bs

```diff
@@ -8508,6 +8527,357 @@ worker=] algorithm:
 
 </div>
 
+## The storage Module ## {#module-storage}
+
+The <dfn export for=modules>storage</dfn> module contains functionality and
+events related to storage.
+
+### Definition ### {#module-storage-definition}
+
+[=Remote end definition=]
+
+<pre class="cddl remote-cddl">
+StorageCommand = (
+  storage.GetCookies //
+  storage.SetCookie //
+  storage.DeleteCookies
+)
+</pre>
+
+[=Local end definition=]
+
+<pre class="cddl local-cddl">
+StorageResult = (
+  storage.GetCookiesResult //
+  storage.SetCookieResult
+)
+</pre>
+
+### Types ### {#module-storage-types}
+
+#### The storage.PartitionKey Type #### {#type-storage-PartitionKey}
+
+[=Remote end definition=] and [=Local end definition=]
+
+<pre class="cddl local-cddl remote-cddl">
+storage.PartitionKey {
+    ? container: text,
+    ? sourceOrigin: text,
+    Extensible,
+}
+</pre>
+
+The <code>storage.PartitionKey</code> type describes namespaces within which the user agent may organize persistent data such as [=cookies=] and local storage.
+
+A [=remote end=] has a [=/set=] of <dfn>storage partition key attributes</dfn> which contains zero or more implementation-defined string values.
+
+A [=remote end=] has a [=/map=] of <dfn>default values for storage partition key attributes</dfn> which contains zero or more implementation-defined entries. Each key must be a member of the [=storage partition key attributes=] which the [=remote end=] does not require as input to [=storage.getCookies=], [=storage.setCookie=], and [=storage.deleteCookies=]; the corresponding value is a the value which the [=remote end=] uses when one is not provided.
+
+<div algorithm>
+To <dfn>expand a storage partition key spec</dfn> given |partition key spec|:
+
+1. If |partition key spec| is null:
+  1. Let |partition key| be an empty [=/map=].
+1. Otherwise:
+  1. Let |partition key| be |partition key spec|.
+1. For each |name| → |default value| in the [=default values for storage partition key attributes=]:
+  1. If |partition key|[|name|] does not [=map/exist=]:
+    1. [=map/Set=] |partition key|[|name|] to |default value|.
+1. Return |partition key|.
+
+</div>
+
+<div algorithm>
+To <dfn>get the cookie store</dfn> given |storage partition key|:
+
+1. If a [=cookie store=] exists for |storage partition key|:
+  1. Return the matching [=cookie store=].
+1. Return [=error=] with [=error code=] [=no such storage partition=].
+
+</div>
+
+<p>The following <dfn>table for cookie conversion</dfn>
+ defines the cookie concepts relevant to WebDriver BiDi,
+ how these are referred to in [[COOKIES]],
+ as well as what keys they map to in a [=serialized cookie=].
```

Comment:
If we want to re-use [the corresponding table in WebDriver Classic](https://w3c.github.io/webdriver/#dfn-table-for-cookie-conversion), we'll have to special-case the "Cookie expiry time" concept because WebDriver classic uses `"expiry"` as the JSON key, but [BiDi's `network.Cookie`](https://w3c.github.io/webdriver-bidi/#type-network-Cookie) uses `"expires"`.

Reply:
I believe it's not too late for changing the [BiDi's `network.Cookie`](https://w3c.github.io/webdriver-bidi/#type-network-Cookie) "expires" to "expiry" to align with WebDriver Classic.

Reply:
Oh, that's great! [Here's a patch for that.](https://github.com/w3c/webdriver-bidi/pull/596)